ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/clientlibs/editor/js/editDialog.js

```diff
@@ -164,14 +166,70 @@
         }
     }
 
+    /**
+     * hides the RTE/plain text field on the basis of rich text checkbox and
+     * resolves it's value.
+     *
+     * @param {HTMLElement} dialog The dialog on which the operation is to be performed.
+     * @param isTitleRichText is rich text checkbox selected.
+     * @param isToggled is this function being called on toggle of rich text checkbox.
+     */
+    function resolveRichText(dialog, isTitleRichText, isToggled) {
+        var title = dialog.find(BASE_TITLE)[0],
+            richTextTitle = dialog.find(BASE_RICH_TEXT_TITLE)[0],
+            richTextTitleDiv = dialog.find("div[name='richTextTitle']")[0];
+        if(isTitleRichText.checked){
+            hideGraniteComponent(title);
+            showGraniteComponent(richTextTitle);
+            showGraniteComponent($(richTextTitle).parent('div.richtext-container').siblings('label'));
```

Comment:
All these should follow BEM. No CSS selector without BEM paradigm

Reply:
removed it